packages/esm-patient-attachments-app/src/attachments/attachments-table-overview.component.tsx

```diff
@@ -106,7 +106,7 @@ const AttachmentsTableOverview: React.FC<AttachmentsTableOverviewProps> = ({
                     <TableCell key={cell.id}>{cell.value?.content ?? cell.value}</TableCell>
                   ))}
                   <TableCell className="cds--table-column-menu">
-                    <OverflowMenu size={isTablet ? 'lg' : 'sm'} flipped>
+                    <OverflowMenu size={isTablet ? 'lg' : 'sm'} flipped enterDelayMs={1000000}>
```

Comment:
What is the underlying issue that is bringing the padding on the bottom?
Thanks!

Reply:
It's the popover that appears on the hover ring of the overflow menu, to fix that since there is no option to turn the popover off, I chose to add a delay.